pandas/tests/series/methods/test_to_dict.py

```diff
@@ -20,3 +21,18 @@ def test_to_dict(self, mapping, datetime_series):
         from_method = Series(datetime_series.to_dict(collections.Counter))
         from_constructor = Series(collections.Counter(datetime_series.items()))
         tm.assert_series_equal(from_method, from_constructor)
+
+    @pytest.mark.parametrize(
```

Comment:
this seems duplicative of the frame one?

Reply:
It's the OP in #25969 - and possibly it's a bit different. It's the presence of the string in the constructor that messes things up (so the first `input` works on current  master but the second/third fail)